### DISCUSSION TITLE
ppocr v4的字符检测模块在使用CPU推理的时候有结果，但是结果是错误的

### DISCUSSION THREAD

**Discussion by dtiny** [PaddlePaddle/PaddleOCR#13300] [Q&A]
#### 问题描述 / Problem Description
ppocr **v4的字符检测模型在使用CPU推理的时候有结果，但是结果是错误的；在同样的检测设置参数情况下，如果改成GPU推理，结果就是正确的；如果换成v3版本的检测模型，CPU和GPU推理结果都是正确的，唯独v4的CPU检测是错误的。**

#### 运行环境 / Runtime Environment
- OS: ubuntu18.04
- Paddle:paddle_inference.tgz 2.5 
GPU | 是 | MKL | 8.2 | CUDA11.2/cuDNN8.2/TensorRT8.0
- PaddleOCR:PaddleOCR release/2.7/deploy/cpp_infer/中的c++推理代码

#### 复现代码 / Reproduction Code
检测的参数设置如下：
use_gpu 0
use_tensorrt 0
use_fp16   1
gpu_mem  4000
cpu_math_library_num_threads  10
use_mkldnn 1
limit_side_len  736
limit_type min
det_db_thresh  0.3
det_db_box_thresh  0.55
det_db_unclip_ratio  2.0
use_polygon_score 1
det_db_score_mode fast
use_dilation 0
det_precision fp16

#### 完整报错 / Complete Error Message
有推理结果，没有异常报错发现。

**Comment by GreatV** [comment]
结果是错误的指的是？

**Reply by dtiny** [reply]
使用CPU推理，推理是有结果的，就是有目标检测框输出的，但是框的位置、大小和数量全是错误，跟实际的字符行不沾边，没有一个四边形框是能够近似落在字符行上面的。
